controller_manager/include/controller_manager/controller_loader_interface.hpp

```diff
@@ -40,6 +42,18 @@ class ControllerLoaderInterface
 
   CONTROLLER_MANAGER_PUBLIC
   virtual bool is_available(const std::string & controller_type) const = 0;
+
+  CONTROLLER_MANAGER_PUBLIC
+  const std::string & getName() const {return name_;}
+
+  CONTROLLER_MANAGER_PUBLIC
+  virtual std::vector<std::string> getDeclaredClasses() = 0;
```

Comment:
Certainly, I'm not too trigger happy with consts on Interface classes, since I don't know what mechanisms will all subclasses need to perform an operation.

But I agree that this is pretty harmless.